src/sentry/migrations/0238_fill_org_onboarding_tasks.py

```diff
@@ -0,0 +1,581 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        # These are constants, not models
+        from sentry.models import OnboardingTask, OnboardingTaskStatus
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        Group = orm['sentry.Group']
+        Organization = orm['sentry.Organization']
+        OrganizationMember = orm['sentry.OrganizationMember']
+        OrganizationOnboardingTask = orm['sentry.OrganizationOnboardingTask']
+        Project = orm['sentry.Project']
+
+        queryset = Organization.objects.all()
+
+        for org in RangeQuerySetWrapperWithProgressBar(queryset):
+            if Project.objects.filter(organization=org).count() > 0:
+                OrganizationOnboardingTask.objects.create(
+                    organization=org,
+                    task=OrganizationOnboardingTask.FIRST_PROJECT,
+                    status=OnboardingTaskStatus.COMPLETE
+                )
+
+            projects = Project.objects.filter(organization=org).exclude(first_event=None).order_by('first_event')
+            project_count = projects.count()
+            if project_count > 0:
+                OrganizationOnboardingTask.objects.create(
+                    organization=org,
+                    task=OrganizationOnboardingTask.FIRST_EVENT,
+                    status=OnboardingTaskStatus.COMPLETE,
+                    project_id=projects[0].id
+                )
+
+                if project_count > 1:
+
+                    platforms = set()
+                    for project in projects:
+                        # Lazily assume project's events are uniformly of the same platform
+                        group = Group.objects.filter(project=project).first()
+                        if group:
+                            platforms.add(group.platform)
+                            if len(platforms) > 1:
+                                OrganizationOnboardingTask.objects.create(
+                                    organization=org,
+                                    task=OrganizationOnboardingTask.SECOND_PLATFORM,
+                                    status=OnboardingTaskStatus.COMPLETE,
+                                    project_id=project.id
+                                )
+                                break
+
+            # INVITE_MEMBER
+            if OrganizationMember.objects.filter(organization=org).exclude(user=None).count() > 1:
```

Comment:
if I'm changing it to `projects = Project.objects.filter(organization=org).exclude(first_event=None).order_by('first_event').all()`, I should just get the len(projects), instead of doing the count.